### PR TITLE
enable/disable comments based on post metainfo

### LINKED ISSUE #23: Add ability to enable/disable comments for a specific post

### PR DESCRIPTION
**Pull request by barbosa**
Closes #23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +73,7 @@
       "prettier --write"
     ],
     "*.md": [
-      "yarn textlint --fix"
+      "yarn textlint"
```
Comment (barbosa) [inline package.json:76]: Too risky keeping this `--fix` here for spell checks.

File: .textlintrc [hunk 2]
```diff
@@ -27,14 +27,12 @@
...
         "/Loadsmart/i",
         "localhost",
+        "Morocho",
         "Netlify"
       ]
     },
     "terminology": {
       "defaultTerms": true
-    },
-    "write-good": {
```
Comment (barbosa) [inline .textlintrc:36]: 'write-good' seems to be too strict. Better leave this off for now.

File: .textlintrc [hunk 3]
```diff
@@ -27,14 +27,12 @@
         "js",
         "/Loadsmart/i",
         "localhost",
+        "Morocho",
```
Comment (davidsonfellipe) [inline .textlintrc:30]: Unbelievable, I do not believe that this lib does not support Morocho. 😂

Comment (barbosa) [reply .textlintrc:30]: @davidsonfellipe `--fix` changed it to `Marocco` 🤦‍♂️

**Pull request merged**